### DISCUSSION TITLE
Community channel

### DISCUSSION THREAD

**Discussion by PrimaIVoid** [spicelang/spice#372] [Ideas]
I think would be a nice idea to create a Discord channel for the Spice language as most other programming languages have, to be able to keep in touch with the developers/contributors and grow up a community and have a quick feedback

**Comment by marcauberer** [comment]
Hey @PrimaIVoid! That is actually a good idea, see https://discord.gg/D6sCsJyWPg
Also added it to the README

**Comment by PrimaIVoid** [comment]
Right now it's not available a "general chat" channel, only the vocal chat.
Also for what i see seem like it could make sense to make two separate "general chat" channels one for english and one for german

**Reply by marcauberer** [reply]
Done
